akka-stream-tests/src/test/scala/akka/stream/scaladsl/SinkSpec.scala

```diff
@@ -228,6 +228,21 @@ class SinkSpec extends StreamSpec with DefaultTimeout with ScalaFutures {
     }
   }
 
+  "The seq sink" must {
+    "collect the streamed elements into a sequence" in {
+      // #seq-operator-example
+      val source = Source(1 to 3)
+      val result = source.runWith(Sink.seq[Int])
```

Comment:
It might make sense to specify the type of `result`. The type parameter on `seq` is less important.

Reply:
Indeed, the type of result is actually pretty nasty and I still don't get its meaning